i18n/fr/docusaurus-plugin-content-docs/current/organization/offices.mdx

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 2
+sidebar_label: "Bureaux"
+hide_table_of_contents: true
+---
+
+# Bureaux 🏢
+
+Nos bureaux sont situés au 1 Passage de l'Europe 31400 Toulouse - FRANCE
+
+8 places de parking sont réservées à OKP4.
+
+Notre filiale OKP4 Gmbh est située  
+Our subsidiary OKP4 GmbH is located  
+Peter-Lühr-Straße 7  
+81739 Munich  
+GERMANY
+
+#### Jetez un coup d'œil à notre bureau de Toulouse !
```

Comment:
To remain consistent with the French text above, I would have kept this translation:
#### Jetez un coup d'œil à nos bureaux de Toulouse !

Reply:
Done!